app/lib/world/world.coffee

```diff
@@ -260,6 +260,14 @@ module.exports = class World
         system.start @thangs
       catch err
         console.error "Error starting system!", system, err
+    # Silently clamp health after systems run.
+    hero = _.find(@thangs, (thang) -> thang.id == 'Hero Placeholder')
+    if hero?
```

Comment:
Should this be refactored out?
It's not long enough and doesn't need to be called anywhere else currently.

Reply:
I would just put it into its own `world` method here called `clampHeroHealth` or something like that.